bench-tps/src/bench_tps_client/rpc_client.rs

```diff
@@ -9,8 +10,17 @@ use {
 
 impl BenchTpsClient for RpcClient {
     fn send_transaction(&self, transaction: Transaction) -> Result<Signature> {
-        RpcClient::send_transaction(self, &transaction).map_err(|err| err.into())
+        RpcClient::send_transaction_with_config(
+            self,
+            &transaction,
+            RpcSendTransactionConfig {
+                preflight_commitment: Some(self.commitment().commitment),
+                ..RpcSendTransactionConfig::default()
+            },
```

Comment:
nit: Since we decided to leave `skip_preflight` alone, I think we can go back to `send_transaction()` right ? `send_transaction()` calls `send_transaction_with_config()` does almost exactly what you're doing here with building up the config struct